styles/globals.css

```diff
@@ -3,4 +3,19 @@
 
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+.content {
```

Comment:
Instead of adding the styling for your component here, you can add styling with tailwind classNames. see the [docs](https://tailwindcss.com)

Reply:
> Also, the links don't follow the figma specifications. change the font weight

Where can I find the figma specifications?

Reply:
Here you go :) https://www.figma.com/file/AT1rNs6YZtkk7jjo5iWMZW/Fiveable-Open-Source-Landing?node-id=0%3A1